packages/fetch/test/request.js

```diff
@@ -338,6 +338,7 @@ describe('Request', () => {
 		ogFormData.append('a', 1);
 		ogFormData.append('b', 2);
 		ogFormData.append('file', new File(['content'], 'file.txt'));
+		ogFormData.append('empty-file', new File([], '', { type: 'application/octet-stream' }));
```

Comment:
might be worth adding a comment/link here indicating this is exactly what happens to empty file inputs when [constructing the form data set](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#constructing-the-form-data-set) (e.g. native form submission, or `new FormData(someForm)`